pyconuk/settings.py

```diff
@@ -0,0 +1,118 @@
+"""
+Django settings for pyconuk project.
+
+Generated by 'django-admin startproject' using Django 1.9.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.9/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.9/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.9/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'secret'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    # These two apps are required for Django to work properly, even though we
+    # don't use them directly.
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+
+    'django.contrib.staticfiles',
+    
+    'django_amber',
+    'markdown_deux',
+
+    'pyconuk',
+]
```

Comment:
Apps should be ordered:
1. Project
2. Third Party
3. Django

The way templates are looked up is first come first served based on this ordering so to have app level overrides they must come first.